src/streamlink/plugins/youtube.py

```diff
@@ -306,14 +244,14 @@ def _get_stream_info(self, url):
             params = {"video_id": video_id}
             params.update(_params)
 
-            res = http.get(API_VIDEO_INFO, params=params, headers=HLS_HEADERS)
-            info_parsed = parse_query(res.text, name="config", schema=_config_schema)
+            res = http.get(self._video_info_url, params=params)
+            info_parsed = parse_query(res.content, name="config", schema=_config_schema)
```

Comment:
`.content` breaks it

```
[cli][debug] Python:     3.6.5
[cli][debug] Streamlink: 0.14.2+14.g0e48e2f.dirty
[cli][debug] Requests(2.19.1), Socks(1.6.7), Websocket(0.48.0)
[cli][info] Found matching plugin youtube for URL https://www.youtube.com/user/france24/live
[plugin.youtube][debug] Re-directing to canonical URL:
https://www.youtube.com/channel/UCCCPCZNChQdGa9EkATeye4g/live
error: 'ascii' codec can't encode character '\xe9' in position 111: ordinal not in range(128)
```

Reply:
Should be fixed with a rebase on master. `.content` needs to be used over `.text` because the data is urlencoded utf8, which is not handled by `.text` as it just decodes it as `utf8` to unicode.